### PR TITLE
Fixed a bug where clicking on a link resulted in incorrect redirection.

### LINKED ISSUE #128: The configure of DebugSlo container list's diagnostic link is not currect.
### There's no existing/similar bug report.

- [X] I have searched the existing issues

### Describe the bug:

 When the user clicks on the link of debugSlo container list's diagnostic link, the browser will open a blank page without going to page debugpod.

### Expected behavior:

When the user clicks on the link, debugpod page  will open and receive the correct parameters

### Additional

_No response_

### Lunettes version

_No response_

### Kubernetes version

_No response_

### PR DESCRIPTION
**Pull request by LLKCoder**
## Description
Fixed a bug where clicking on a link resulted in incorrect redirection.

## New Configuretion
**DebugSlo**

**podyaml**

**nodeyaml**

Close #128

### CODE REVIEW

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml [hunk 1]
```diff
@@ -697,7 +697,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "NodeYaml",
-                        "url": "/d/nodeyamllinks/nodeyaml?orgId=1&var-nodeName=${__data.fields[\"NodeName\"]}"
+                        "url": "/d/nodeyamllinks/nodeyaml?var-nodeName=${__data.fields[\"NodeName\"]}"
```
Comment (linuzb) [inline deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:700]: Is there any difference in the usage of `__data.fields[\"NodeName\"]` and `__data.fields.PodUid`? It is recommended to unify the format.

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml [hunk 2]
```diff
@@ -2609,7 +2609,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "Pod信息",
-                        "url": "\r\nhttp://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpod?uid=${__data.fields.PodUid}"
+                        "url": "\r\nd/lunettes-debugpod/lunettes-debugpod?var-podinfo=uid&var-podinfovalue=${__data.fields.PodUid}"
```
Comment (linuzb) [inline deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:2612]: Can the `\r\n` be removed? This is supposed to be a new line character in Windows, which I don't think is needed here.

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml [hunk 3]
```diff
@@ -2609,7 +2609,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "Pod信息",
-                        "url": "\r\nhttp://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpod?uid=${__data.fields.PodUid}"
```
Comment (d3c3mber) [inline deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:2612]: there's also a few other places using `lunettes.lunettes.svc.cluster.local:8080`, can you please help to update those links as well @LLKCoder

Comment (LLKCoder) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:2612]: okay

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml (hunk 2 above)
Comment (LLKCoder) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:2612]: okay

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml [hunk 4]
```diff
@@ -672,7 +672,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "yaml",
-                        "url": "/d/podyamllinks/podyaml?orgId=1&var-uid=${__data.fields[\"PodUID\"]}"
+                        "url": "/d/podyamllinks/podyaml?var-uid=${__data.fields[\"PodUID\"]}"
```
Comment (linuzb) [inline deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:675]: Please unify the `__data.fields[\"PodUID\"]` format.

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml (hunk 1 above)
Comment (LLKCoder) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml]: All have been unified into the format of 【 __data.fields.PodUid 】

**Review by d3c3mber — APPROVED** [review]

**Review by hnhbwlp — APPROVED** [review]
lgtm

**Pull request merged**